data_rentgen/consumer/extractors/dataset.py

```diff
@@ -64,6 +57,52 @@ def extract_dataset(dataset: OpenLineageDataset | OpenLineageSymlinkIdentifier)
     )
 
 
+def extract_io_dataset(dataset: OpenLineageDataset) -> tuple[DatasetDTO, list[DatasetSymlinkDTO]]:
+    if dataset.facets.symlinks:
+        table_symlinks = [
+            identifier
+            for identifier in dataset.facets.symlinks.identifiers
+            if identifier.type == OpenLineageSymlinkType.TABLE
+        ]
+        if table_symlinks:
+            # TODO: add support for multiple TABLE symlinks
```

Comment:
It's better to add comment why we do this in the first place. Like https://github.com/OpenLineage/OpenLineage/issues/2718